src/lib/journey/callbacks/terms-and-conditions/terms-conditions.svelte

```diff
@@ -40,5 +47,8 @@
 </script>
 
 <Checkbox {firstInvalidInput} key={inputName} onChange={setValue} value={false}>
-  <T key="termsAndConditions" />
+  <span>
```

Comment:
Is the `span` necessary?

Reply:
I had hoped not, but it seems to affect the formatting of the link if I remove it:

<img width="438" alt="Screenshot 2022-10-19 at 12 59 48" src="https://user-images.githubusercontent.com/30521218/196684882-1979225a-7f25-464d-8f00-eefcc9fb4107.png">

Maybe there's something we can do with CSS for this if you think it's preferable to removing the span?

Reply:
Oh, yeah, hmm ... let's add an optional `classes` prop to the `Message` component, so we can pass in a few extra classes for optional styling purposes. Here are the classes you'd want to pass in:

```
tw_col-start-2 tw_row-start-2
```

These classes just instruct the `Message` component to render in a different block of the grid used for the `Checkbox` component.